components/Workspaces/TrackedRepoWizard/PasteReposStep.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import Button from "components/atoms/Button/button";
+
+interface PasteReposStepProps {
+  onBulkAddRepos: (repos: string[]) => void;
+}
+
+export const PasteReposStep = ({ onBulkAddRepos }: PasteReposStepProps) => {
+  const [pastedInput, setPastedInput] = useState("");
+
+  const parseInput = () => {
```

Comment:
refactor (blocking): if the textarea is empty return instead of doing the bulk import. Otherwise you get to the next step with a repository that is an empty string.

![CleanShot 2024-02-07 at 09 51 22](https://github.com/open-sauced/app/assets/833231/30d11706-6f70-4d9a-919e-92105ce4a654)

Until the value of the textarea has text, the import button should be disabled.